lib/kino/explorer.ex

```diff
@@ -205,7 +187,11 @@ defmodule Kino.Explorer do
     |> DataFrame.to_columns()
   end
 
-  defp compute_summaries?(series, exp_ver_0_7_2_gte?) do
+  defp compute_summaries?(series) do
+    # hacky way to provide backward compatibility for {:list, numeric} error
+    # https://github.com/elixir-explorer/explorer/issues/787
+    exp_ver_0_7_2_gte? = Explorer.Shared.dtypes() |> Enum.member?({:s, 8})
```

Comment:
As far as I understand it's actually `exp_ver_0_7_2_gt?`, because `{:s, 8}` is only on main, right?

Also, let's add a TODO to remove the check once we require only explorer v0.8.